sitewit/models.py

```diff
@@ -192,13 +192,14 @@ def set_site_id(cls, account_token, new_site_id):
         return Account(result)
 
     @classmethod
-    def _get_valid_user_name(cls, user_name):
+    def _get_valid_user_name(cls, user):
```

Comment:
>DISCLAIMER: It is just a small general note. It is not 100% applicable to the current changes. 

IMO, usually, it is not a good idea to make a function dependent on the whole object when only one (or two) object's properties are needed. 

Some of the cons are:
- We need to mock (or get from a database) model instances when testing. 
- It can cause problems when we have the sufficient data but there is no model instance. 
- Another one point is code reusability. The function which operates with a name does not always need to know about `user` model and it's methods.

Reply:
i'm too slow, it has been merged :snail:

Reply:
Completely agree with your points.

Reply:
It is an internal helper method, so I don't think it's so important.